x/bank/keeper/keeper_test.go

```diff
@@ -89,17 +99,43 @@ func (suite *IntegrationTestSuite) SetupTest() {
 }
 
 func (suite *IntegrationTestSuite) TestSupply() {
-	app, ctx := suite.app, suite.ctx
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{Height: 1})
+	appCodec := simapp.MakeTestEncodingConfig().Marshaler
+
+	require := suite.Require()
+
+	// add module accounts to supply keeper
+	maccPerms := getModuleAccPerms()
+	authKeeper := authkeeper.NewAccountKeeper(
```

Comment:
I think you can add authKeepr and bank keeper constructor to the same function - it's always used and repeated next to the maccPerms.